proposals/working-group/Comparisons.adoc

```diff
@@ -0,0 +1,36 @@
+= Pros/Cons of Each Proposal
+
+== MPWG
```

Comment:
Let's spell these out so that's it very clear for the casual reader which proposal is which...  In this case, MicroProfile Working Group.